app/services/NinoLookupService.scala

```diff
@@ -27,17 +27,17 @@ import scala.concurrent.ExecutionContext.Implicits.global
 import scala.concurrent.Future
 
 @Singleton
-class NinoLookupService @Inject()(val ninoLookupConnector: NinoLookupConnector) {
+class NinoLookupService @Inject()(val ninoLookupConnector: IncomeSourceDetailsConnector) {
```

Comment:
rather than call this `ninoLookupConnector` change to `incomeSourceDetailsConnector`